### PR TITLE
Test utils visbrain

### PR DESCRIPTION
**Pull request by davidmeunier79**

### CODE REVIEW

File: graphpype/tests/test_utils_visbrain.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    #data_path, "data_graph", "Z_List-info_nodes.txt")
+
+
+#from graphpype.utils_net import (return_net_list, read_lol_file,
+                                 #read_Pajek_corres_nodes, read_Pajek_corres_nodes_and_sparse_matrix,
+                                 #compute_modular_matrix, get_strength_values_from_info_nodes_file,
+                                 #get_strength_pos_values_from_info_nodes_file, get_strength_neg_values_from_info_nodes_file,
+                                 #get_degree_pos_values_from_info_nodes_file, get_degree_neg_values_from_info_nodes_file,
+                                 #export_Louvain_net_from_list)
+
```
Comment (jasmainak) [inline graphpype/tests/test_utils_visbrain.py]: you need to remove code comments ...

File: graphpype/tests/test_utils_visbrain.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+################################################ test if vispy and visbrain package are available
+
+def test_vispy():
```
Comment (jasmainak) [inline graphpype/tests/test_utils_visbrain.py]: docstring missing

File: graphpype/tests/test_utils_visbrain.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    except ImportError:
+        print ("vispy not installed")
+        
+def test_visbrain():
+    
+    try:
+        import visbrain
+        
+    except ImportError:
+        print ("visbrain not installed")
```
Comment (jasmainak) [inline graphpype/tests/test_utils_visbrain.py]: this line should go inside `graphypype` -- not in the tests

File: graphpype/tests/test_utils_visbrain.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                                 #get_strength_pos_values_from_info_nodes_file, get_strength_neg_values_from_info_nodes_file,
+                                 #get_degree_pos_values_from_info_nodes_file, get_degree_neg_values_from_info_nodes_file,
+                                 #export_Louvain_net_from_list)
+
+################################################ test if vispy and visbrain package are available
+
+def test_vispy():
+    
+    try:
+        import vispy
```
Comment (EtienneCmb) [inline graphpype/tests/test_utils_visbrain.py]: I don't think you need to test vispy installation as it's already tested when Visbrain is imported

**Comment by davidmeunier79** [comment]
@EtienneCmb can you check what needs to be changed for Appveyor and Travis so the build will not crashed?

File: graphpype/tests/test_utils_visbrain.py (hunk 4 above)
Comment (davidmeunier79) [reply graphpype/tests/test_utils_visbrain.py]: @EtienneCmb agreed but my point was to see if the lines I added to the continuous integration config. files would be enough to import vispy. It is working with vispy but not with visbrain

File: .travis.yml [hunk 5]
```diff
@@ -39,10 +39,14 @@ before_install:
...
     # Create the python environment ;
     - conda create -q -n testenv python=$TRAVIS_PYTHON_VERSION numpy pip cython matplotlib;
+    - conda install pyqt
     - source activate testenv
     # Testing dependencies
     - pip install codecov pytest pytest-cov;
     - pip install neuropycon_data
+    - pip install git+https://github.com/vispy/vispy.git
+    - pip install pandas PyOpenGL PyOpenGL_accelerate
+    - pip install visbrain
```
Comment (jasmainak) [inline .travis.yml:49]: how come you don't need `radatools` in `travis`?

Comment (davidmeunier79) [reply .travis.yml:49]: > how come you don't need `radatools` in `travis`?

Yes it is true radatools should be installed in travis, but so far the nodes in interface/radatools are not tested yet as requiring the nipype engine to work, and for that the appveyor should be modified. 
Once the other PR is solved I will give a go at installing Radatools on travis

**Pull request closed without merging**